src/types/Calculator.ts

```diff
@@ -177,6 +177,8 @@ export interface StatementResult {
   raw?: any;
   /** Pretty result for this statement. */
   pretty?: string;
+  /** TRUE if the statement is a function */
+  isFunction?: 'TRUE' | 'FALSE';
```

Comment:
Okay- I gotta know. Why is this a set of possible strings rather than just a boolean?